src/Umbraco.Web/Install/InstallHelper.cs

```diff
@@ -23,6 +23,7 @@ namespace Umbraco.Web.Install
 {
     internal class InstallHelper
     {
+        private static readonly HttpClient HttpClient = new HttpClient();

```

Comment:
Same as the previous comment, it's only used in `GetStarterKits` so let's only initialize it there.